app/views/donations/new.html.erb

```diff
@@ -14,15 +20,42 @@
 
     <%= radio_button_tag :monthly, false %>
     <%= label_tag :monthly_false, "Charge me once" %>
-
   </article>
 
-  <script src="https://checkout.stripe.com/checkout.js" class="stripe-button"
-          data-key="<%= Rails.configuration.stripe[:publishable_key] %>"
-          data-description="A month's subscription"
-          data-amount="5000"
-          data-zip-code="true"
-          data-locale="auto"></script>
-<% end %>
+  <button id="js-stripe-button">DONATE</button>
+
+  <script>
+    var handler = StripeCheckout.configure({
```

Comment:
If anyone has any ideas as to how to get this out of the view, I'm _all ears_. The problem is that the:
```
<script src="https://checkout.stripe.com/checkout.js"></script>
```
Near the top of this file has already loaded and allows for calling `StripeCheckout.configure`. I'm not sure how to extract everything in the `<script>` block out into the `donations.js` while still allowing for access to `StripeCheckout`.